### ISSUE TITLE
[REVIEW]: COINSTAC: Collaborative Informatics and Neuroimaging Suite Toolkit for Anonymous Computation

### ISSUE THREAD

**Issue by whedon** [openjournals/joss-reviews#2166]
**Submitting author:** @hvgazula (<a href="http://orcid.org/0000-0001-9318-8871">Harshvardhan Gazula</a>)
**Repository:** <a href="https://github.com/trendscenter/coinstac" target ="_blank">https://github.com/trendscenter/coinstac</a>
**Version:** v5.1.3
**Editor:** @cMadan
**Reviewer:** @gkiar, @yarikoptic
**Archive:** <a href="https://doi.org/10.5281/zenodo.4111006" target="_blank">10.5281/zenodo.4111006</a>

## Status

[![status](https://joss.theoj.org/papers/c51a4a8f4aec5d8af4c8c31d2a37b0ad/status.svg)](https://joss.theoj.org/papers/c51a4a8f4aec5d8af4c8c31d2a37b0ad)

Status badge code:

```
HTML: <a href="https://joss.theoj.org/papers/c51a4a8f4aec5d8af4c8c31d2a37b0ad"><img src="https://joss.theoj.org/papers/c51a4a8f4aec5d8af4c8c31d2a37b0ad/status.svg"></a>
Markdown: [![status](https://joss.theoj.org/papers/c51a4a8f4aec5d8af4c8c31d2a37b0ad/status.svg)](https://joss.theoj.org/papers/c51a4a8f4aec5d8af4c8c31d2a37b0ad)
```
**Reviewers and authors**:

Please avoid lengthy details of difficulties in the review thread. Instead, please create a new issue in the <a href="https://github.com/MRN-Code/coinstac" target="_blank">target repository</a> and link to those issues (especially acceptance-blockers) by leaving comments in the review thread below. (For completists: if the target issue tracker is also on GitHub, linking the review thread in the issue or vice versa will create corresponding breadcrumb trails in the link target.)

## Reviewer instructions & questions

@gkiar & @yarikoptic, please carry out your review in this issue by updating the checklist below. If you cannot edit the checklist please:
1. Make sure you're logged in to your GitHub account
2. Be sure to accept the invite at this URL: https://github.com/openjournals/joss-reviews/invitations

The reviewer guidelines are available here: https://joss.readthedocs.io/en/latest/reviewer_guidelines.html. Any questions/concerns please let @cMadan know.

✨ **Please try and complete your review in the next two weeks** ✨ 

### Conflict of interest

- [x] I confirm that I have read the [JOSS conflict of interest (COI) policy](https://github.com/openjournals/joss/blob/master/COI.md) and that: I have no COIs with reviewing this work or that any perceived COIs have been waived by JOSS for the purpose of this review.

### Code of Conduct

- [x] I confirm that I read and will adhere to the [JOSS code of conduct](https://joss.theoj.org/about#code_of_conduct).

### General checks

- [x] **Repository:** Is the source code for this software available at the <a target="_blank" href="https://github.com/MRN-Code/coinstac">repository url</a>?
- [x] **License:** Does the repository contain a plain-text LICENSE file with the contents of an [OSI approved](https://opensource.org/licenses/alphabetical) software license?
- [x] **Contribution and authorship:** Has the submitting author (@hvgazula) made major contributions to the software? Does the full list of paper authors seem appropriate and complete?

### Functionality

- [x] **Installation:** Does installation proceed as outlined in the documentation?
- [ ] **Functionality:** Have the functional claims of the software been confirmed?
- [x] **Performance:** If there are any performance claims of the software, have they been confirmed? (If there are no claims, please check off this item.)

### Documentation

- [x] **A statement of need:** Do the authors clearly state what problems the software is designed to solve and who the target audience is?
- [x] **Installation instructions:** Is there a clearly-stated list of dependencies? Ideally these should be handled with an automated package management solution.
- [ ] **Example usage:** Do the authors include examples of how to use the software (ideally to solve real-world analysis problems).
- [ ] **Functionality documentation:** Is the core functionality of the software documented to a satisfactory level (e.g., API method documentation)?
- [x] **Automated tests:** Are there automated tests or manual steps described so that the functionality of the software can be verified?
- [ ] **Community guidelines:** Are there clear guidelines for third parties wishing to 1) Contribute to the software 2) Report issues or problems with the software 3) Seek support

### Software paper

- [x] **Summary:** Has a clear description of the high-level functionality and purpose of the software for a diverse, non-specialist audience been provided?
- [x] **A statement of need:** Do the authors clearly state what problems the software is designed to solve and who the target audience is?
- [x] **State of the field:** Do the authors describe how this software compares to other commonly-used packages?
- [x] **Quality of writing:** Is the paper well written (i.e., it does not require editing for structure, language, or writing quality)?
- [x] **References:** Is the list of references complete, and is everything cited appropriately that should be cited (e.g., papers, datasets, software)? Do references in the text use the proper [citation syntax]( https://rmarkdown.rstudio.com/authoring_bibliographies_and_citations.html#citation_syntax)?

### Conflict of interest

- [x] I confirm that I have read the [JOSS conflict of interest (COI) policy](https://github.com/openjournals/joss/blob/master/COI.md) and that: I have no COIs with reviewing this work or that any perceived COIs have been waived by JOSS for the purpose of this review.

### Code of Conduct

- [x] I confirm that I read and will adhere to the [JOSS code of conduct](https://joss.theoj.org/about#code_of_conduct).

### General checks

- [x] **Repository:** Is the source code for this software available at the <a target="_blank" href="https://github.com/MRN-Code/coinstac">repository url</a>? (correction: moved and being redirected to https://github.com/trendscenter/coinstac now)
- [x] **License:** Does the repository contain a plain-text LICENSE file with the contents of an [OSI approved](https://opensource.org/licenses/alphabetical) software license? (needs work: https://github.com/trendscenter/coinstac/issues/879, https://github.com/trendscenter/coinstac/issues/876)
- [x] **Contribution and authorship:** Has the submitting author (@hvgazula) made major contributions to the software? Does the full list of paper authors seem appropriate and complete?

### Functionality

- [x] **Installation:** Does installation proceed as outlined in the documentation?
- [x] **Functionality:** Have the functional claims of the software been confirmed?
- [x] **Performance:** If there are any performance claims of the software, have they been confirmed? (If there are no claims, please check off this item.)

### Documentation

- [x] **A statement of need:** Do the authors clearly state what problems the software is designed to solve and who the target audience is?
- [x] **Installation instructions:** Is there a clearly-stated list of dependencies? Ideally these should be handled with an automated package management solution.
- [x] **Example usage:** Do the authors include examples of how to use the software (ideally to solve real-world analysis problems).
- [x] **Functionality documentation:** Is the core functionality of the software documented to a satisfactory level (e.g., API method documentation)?
- [ ] **Automated tests:** Are there automated tests or manual steps described so that the functionality of the software can be verified?
- [x] **Community guidelines:** Are there clear guidelines for third parties wishing to 1) Contribute to the software 2) Report issues or problems with the software 3) Seek support

### Software paper

- [x] **Summary:** Has a clear description of the high-level functionality and purpose of the software for a diverse, non-specialist audience been provided?
- [x] **A statement of need:** Do the authors clearly state what problems the software is designed to solve and who the target audience is?
- [x] **State of the field:** Do the authors describe how this software compares to other commonly-used packages?
- [x] **Quality of writing:** Is the paper well written (i.e., it does not require editing for structure, language, or writing quality)?
- [x] **References:** Is the list of references complete, and is everything cited appropriately that should be cited (e.g., papers, datasets, software)? Do references in the text use the proper [citation syntax]( https://rmarkdown.rstudio.com/authoring_bibliographies_and_citations.html#citation_syntax)?

**Comment by whedon** [comment]
Hello human, I'm @whedon, a robot that can help you with some common editorial tasks. @gkiar, @yarikoptic it looks like you're currently assigned to review this paper :tada:.

**:star: Important :star:**

If you haven't already, you should seriously consider unsubscribing from GitHub notifications for this (https://github.com/openjournals/joss-reviews) repository. As a reviewer, you're probably currently watching this repository which means for GitHub's default behaviour you will receive notifications (emails) for all reviews 😿

To fix this do the following two things:

1. Set yourself as 'Not watching' https://github.com/openjournals/joss-reviews:

![watching](https://cloud.githubusercontent.com/assets/4483/20250593/64d7ce48-a9de-11e6-9225-d3dfb3e48e68.png)

2. You may also like to change your default settings for this watching repositories in your GitHub profile here: https://github.com/settings/notifications

![notifications](https://cloud.githubusercontent.com/assets/4483/20250604/79eb3144-a9de-11e6-9e51-0fe940966689.png)


For a list of things I can do to help you, just type:

```
@whedon commands
```

For example, to regenerate the paper pdf after making changes in the paper's md or bib files, type:

```
@whedon generate pdf
```

**Comment by whedon** [comment]
PDF failed to compile for issue #2166 with the following error: 

 Can't find any papers to compile :-(

**Comment by gkiar** [comment]
@whedon generate pdf from branch paper-branch

**Comment by whedon** [comment]
```
Attempting PDF compilation from custom branch paper-branch. Reticulating splines etc...
```

**Comment by whedon** [comment]
[ :point_right: Check article proof :page_facing_up: :point_left: ](https://github.com/openjournals/joss-papers/blob/joss.02166/joss.02166/10.21105.joss.02166.pdf)

**Comment by gkiar** [comment]
@cMadan I have left my initial review at https://github.com/trendscenter/coinstac/issues/849 and checked off the already-met requirements in my form above.

**Comment by gkiar** [comment]
@whedon generate pdf from branch paper-branch

**Comment by whedon** [comment]
```
Attempting PDF compilation from custom branch paper-branch. Reticulating splines etc...
```

**Comment by whedon** [comment]
[ :point_right: Check article proof :page_facing_up: :point_left: ](https://github.com/openjournals/joss-papers/blob/joss.02166/joss.02166/10.21105.joss.02166.pdf)

**Comment by arfon** [comment]
:wave: @yarikoptic - just a friendly check-in to see how things are going with your review?

**Comment by yarikoptic** [comment]
@arfon sorry for the delay... will do today/tomorrow

**Comment by yarikoptic** [comment]
Sorry for the delay again, an update: my feedback to authors and here is starting to flow! ;-)

**Comment by yarikoptic** [comment]
FTR: my review of the manuscript with some additional references -- https://github.com/trendscenter/coinstac/issues/881

**Comment by kthyng** [comment]
Hi @hvgazula! Looks like you have a review in as noted immediately above. How are things going on your end to address the comments?

**Comment by hvgazula** [comment]
Hello @kthyng Thanks for checking. I shall be pasting my comments sometime tonight.

**Comment by hvgazula** [comment]
> Hello @kthyng Thanks for checking. I shall be pasting my comments sometime tonight.

Hi @kthyng and @cMadan FYI, I submitted my responses to the second reviewer as well last week and will not wait to hear from both the reviewers if they have any comments. Thanks!

**Comment by hvgazula** [comment]
Tagging @gkiar and @yarikoptic just in case as a reminder 👍

**Comment by cMadan** [comment]
@gkiar @yarikoptic, do you know when you might be able to look over these changes? Thanks!

**Comment by cMadan** [comment]
Fyi. I just sent an email to remind @gkiar and @yarikoptic about this review.

**Comment by hvgazula** [comment]
> Fyi. I just sent an email to remind @gkiar and @yarikoptic about this review.

Thank you very much.

**Comment by gkiar** [comment]
I have updated my review. The only remaining piece from my perspective is the addition of issue guidelines and PR templates for contributors and users, fitting the description provided in the review template above.

**Comment by cMadan** [comment]
Great, thank you, @gkiar!

**Comment by hvgazula** [comment]
Thank you @cMadan and @gkiar. As suggested, the templates have been put in place now. Yet to hear from @yarikoptic.

**Comment by yarikoptic** [comment]
@whedon generate pdf from branch paper-branch

**Comment by whedon** [comment]
```
Attempting PDF compilation from custom branch paper-branch. Reticulating splines etc...
```

**Comment by whedon** [comment]
[ :point_right: Check article proof :page_facing_up: :point_left: ](https://raw.githubusercontent.com/openjournals/joss-papers/joss.02166/joss.02166/10.21105.joss.02166.pdf)

**Comment by yarikoptic** [comment]
I am satisfied with edits to the paper, and have nothing you add to @gkiar 's comments.

**Comment by hvgazula** [comment]
> I am satisfied with edits to the paper, and have nothing you add to @gkiar 's comments.

Thank you very much, @yarikoptic! With the templates too in place as @gkiar suggested, probably it's now up to @cMadan to decide on the next steps.

**Comment by cMadan** [comment]
@yarikoptic, thanks for taking another look! Could you please update the checklist in the first issue? Thanks!

**Comment by yarikoptic** [comment]
Done

**Comment by cMadan** [comment]
@yarikoptic, thank you again for reviewing this submission!

@hvgazula, I have to do some final checks now, but we're almost done!

**Comment by cMadan** [comment]
@whedon generate pdf

**Comment by whedon** [comment]
PDF failed to compile for issue #2166 with the following error: 

 Can't find any papers to compile :-(

**Comment by cMadan** [comment]
@whedon generate pdf from branch paper-branch

**Comment by whedon** [comment]
```
Attempting PDF compilation from custom branch paper-branch. Reticulating splines etc...
```

**Comment by whedon** [comment]
[ :point_right: Check article proof :page_facing_up: :point_left: ](https://raw.githubusercontent.com/openjournals/joss-papers/joss.02166/joss.02166/10.21105.joss.02166.pdf)

**Comment by cMadan** [comment]
@hvgazula, can you fix the bib so that the capitalisation of named programs is correct in the reference list? (E.g., ENIGMA, MCIC, VBM, etc.)

**Comment by cMadan** [comment]
@whedon check references

**Comment by cMadan** [comment]
@whedon check references from branch paper-branch

**Comment by whedon** [comment]
```
Attempting to check references... from custom branch paper-branch
```

**Comment by whedon** [comment]
```
Reference check summary (note 'MISSING' DOIs are suggestions that need verification):

OK DOIs

- 10.1101/846386 is OK

MISSING DOIs

- 10.3389/fnins.2016.00365 may be a valid DOI for title: COINSTAC: a privacy enabled model and prototype for leveraging and processing decentralized brain imaging data
- 10.12688/f1000research.12353.1 may be a valid DOI for title: COINSTAC: Decentralizing the future of brain imaging analysis
- 10.3389/fninf.2018.00055 may be a valid DOI for title: Decentralized analysis of brain imaging data: Voxel-based morphometry and dynamic functional network connectivity
- 10.3389/fninf.2014.00054 may be a valid DOI for title: CBRAIN: a web-based, distributed computing platform for collaborative neuroimaging research
- 10.3389/fninf.2011.00037 may be a valid DOI for title: LORIS: a web-based data management system for multi-center studies
- 10.1016/j.neuroimage.2015.06.076 may be a valid DOI for title: XNAT Central: Open sourcing imaging research data
- 10.1561/9781601988195 may be a valid DOI for title: The algorithmic foundations of differential privacy
- 10.1093/ije/dyv193 may be a valid DOI for title: ViPAR: a software platform for the Virtual Pooling and Analysis of Research Data

INVALID DOIs

- None
```

**Comment by hvgazula** [comment]
> @hvgazula, can you fix the bib so that the capitalisation of specific names are correct in the reference list? (E.g., ENIGMA, MCIC, VBM, etc.)

Thank you @cMadan. You are right. Indeed, the capitalizations are missing ENIGMA, MCIC and VBM in the reference list. It seems that if a title starts with all upper case it works fine. However, an all uppercase in the middle of the title is not retained in the final output. Please advise how I can address this.

**Comment by yarikoptic** [comment]
Enclose within `{}`, so `{ENIGMA}` which will retain capitalization

**Comment by hvgazula** [comment]
> Enclose within `{}`, so `{ENIGMA}` which will retain capitalization

Worked like a charm. Thank you very much for the suggestion, @yarikoptic.

**Comment by hvgazula** [comment]
> > @hvgazula, can you fix the bib so that the capitalisation of specific names are correct in the reference list? (E.g., ENIGMA, MCIC, VBM, etc.)
> 
> Thank you @cMadan. You are right. Indeed, the capitalizations are missing ENIGMA, MCIC and VBM in the reference list. It seems that if a title starts with all upper case it works fine. However, an all uppercase in the middle of the title is not retained in the final output. Please advise how I can address this.

@cMadan The capitalization has been taken care of.

**Comment by cMadan** [comment]
@hvgazula, can you also add DOIs for all bib entries? See the recent comment from whedon for suggestions (but please double check these yourself!)

**Comment by cMadan** [comment]
@hvgazula, just a reminder that the review is currently waiting on your response right now.

**Comment by hvgazula** [comment]
@whedon generate pdf

**Comment by whedon** [comment]
PDF failed to compile for issue #2166 with the following error: 

 Can't find any papers to compile :-(

**Comment by hvgazula** [comment]
@whedon generate pdf from branch paper-branch

**Comment by whedon** [comment]
```
Attempting PDF compilation from custom branch paper-branch. Reticulating splines etc...
```

**Comment by whedon** [comment]
:point_right::page_facing_up: [Download article proof](https://raw.githubusercontent.com/openjournals/joss-papers/joss.02166/joss.02166/10.21105.joss.02166.pdf) :page_facing_up: [View article proof on GitHub](https://github.com/openjournals/joss-papers/blob/joss.02166/joss.02166/10.21105.joss.02166.pdf) :page_facing_up: :point_left:

**Comment by hvgazula** [comment]
@whedon check references from branch paper-branch

**Comment by whedon** [comment]
```
Attempting to check references... from custom branch paper-branch
```

**Comment by whedon** [comment]
```
Reference check summary (note 'MISSING' DOIs are suggestions that need verification):

OK DOIs

- 10.3389/fnins.2016.00365 is OK
- 10.12688/f1000research.12353.1 is OK
- 10.3389/fninf.2018.00055 is OK
- 10.3389/fninf.2014.00054 is OK
- 10.3389/fninf.2011.00037 is OK
- 10.1016/j.neuroimage.2015.06.076 is OK
- 10.7490/f1000research.1114354.1 is OK
- 10.1561/9781601988195 is OK
- 10.1007/s11682-013-9269-5 is OK
- 10.1093/ije/dyv193 is OK
- 10.1101/846386 is OK

MISSING DOIs

- None

INVALID DOIs

- None
```

**Comment by hvgazula** [comment]
> @hvgazula, just a reminder that the review is currently waiting on your response right now.

@cMadan The DOIs have been taken care of. Apologies for taking this long.

**Comment by hvgazula** [comment]
https://github.com/trendscenter/coinstac/releases/tag/v5.1.3

> @hvgazula, everything looks good to me!
> 
> To move forward with accepting your submission, there are a few last things to take care of:
> 
> * [ ]  Make a tagged release of your software, and list the version tag of the archived version here.
5.1.3 https://github.com/trendscenter/coinstac/releases/tag/v5.1.3
> * [ ]  Archive the reviewed software in Zenodo
Done
> * [ ]  Check the Zenodo deposit has the correct metadata, this includes the title (should match the paper title) and author list (make sure the list is correct and people who only made a small fix are not on it); you may also add the authors' ORCID.
Done
> * [ ]  List the Zenodo DOI of the archived version here.
10.5281/zenodo.4111006
> 
> You may find this helpful: https://guides.github.com/activities/citable-code/

**Comment by hvgazula** [comment]
@cMadan I finished all the steps you've asked for. Please let me know if there's anything else I can do. Thank you very much.

**Comment by cMadan** [comment]
@whedon set v5.1.3 as version

**Comment by whedon** [comment]
OK. v5.1.3 is the version.

**Comment by cMadan** [comment]
@whedon set 10.5281/zenodo.4111006 as archive

**Comment by whedon** [comment]
OK. <a href="https://doi.org/10.5281/zenodo.4111006" target="_blank">10.5281/zenodo.4111006</a> is the archive.

**Comment by cMadan** [comment]
@whedon generate pdf

**Comment by whedon** [comment]
PDF failed to compile for issue #2166 with the following error: 

 Can't find any papers to compile :-(

**Comment by cMadan** [comment]
@whedon generate pdf from branch paper-branch

**Comment by whedon** [comment]
```
Attempting PDF compilation from custom branch paper-branch. Reticulating splines etc...
```

**Comment by whedon** [comment]
:point_right::page_facing_up: [Download article proof](https://raw.githubusercontent.com/openjournals/joss-papers/joss.02166/joss.02166/10.21105.joss.02166.pdf) :page_facing_up: [View article proof on GitHub](https://github.com/openjournals/joss-papers/blob/joss.02166/joss.02166/10.21105.joss.02166.pdf) :page_facing_up: :point_left:

**Comment by cMadan** [comment]
@whedon accept

**Comment by whedon** [comment]
```
Attempting dry run of processing paper acceptance...
```

**Comment by whedon** [comment]
PDF failed to compile for issue #2166 with the following error: 

 Can't find any papers to compile :-(

**Comment by cMadan** [comment]
@whedon accept from branch paper-branch

**Comment by whedon** [comment]
```
Attempting dry run of processing paper acceptance...
```

**Comment by whedon** [comment]
```
Reference check summary (note 'MISSING' DOIs are suggestions that need verification):

OK DOIs

- 10.3389/fnins.2016.00365 is OK
- 10.12688/f1000research.12353.1 is OK
- 10.3389/fninf.2018.00055 is OK
- 10.3389/fninf.2014.00054 is OK
- 10.3389/fninf.2011.00037 is OK
- 10.1016/j.neuroimage.2015.06.076 is OK
- 10.7490/f1000research.1114354.1 is OK
- 10.1561/9781601988195 is OK
- 10.1007/s11682-013-9269-5 is OK
- 10.1093/ije/dyv193 is OK
- 10.1101/846386 is OK

MISSING DOIs

- None

INVALID DOIs

- None
```

**Comment by whedon** [comment]
:wave: @openjournals/joss-eics, this paper is ready to be accepted and published.

 Check final proof :point_right: https://github.com/openjournals/joss-papers/pull/1849

If the paper PDF and Crossref deposit XML look good in https://github.com/openjournals/joss-papers/pull/1849, then you can now move forward with accepting the submission by compiling again with the flag `deposit=true` e.g.
 ```
@whedon accept deposit=true from branch paper-branch 
```

**Comment by kyleniemeyer** [comment]
Hi @hvgazula, I'm the EIC on duty this week, doing some final checks before publishing your article.

The only thing I noticed is that your article is missing an explicit [Statement of Need section](https://joss.readthedocs.io/en/latest/submitting.html#what-should-my-paper-contain), which we have begun requiring for all articles. Can you add this? You might be able to just rename an existing section, or move some of your text into the new one.

**Comment by hvgazula** [comment]
> Hi @hvgazula, I'm the EIC on duty this week, doing some final checks before publishing your article.
> 
> The only thing I noticed is that your article is missing an explicit [Statement of Need section](https://joss.readthedocs.io/en/latest/submitting.html#what-should-my-paper-contain), which we have begun requiring for all articles. Can you add this? You might be able to just rename an existing section, or move some of your text into the new one.

Done. Thanks for letting me know about it. Really appreciate it.

**Comment by kyleniemeyer** [comment]
@whedon generate pdf

**Comment by whedon** [comment]
PDF failed to compile for issue #2166 with the following error: 

 Can't find any papers to compile :-(

**Comment by kyleniemeyer** [comment]
@whedon generate pdf from branch paper-branch

**Comment by whedon** [comment]
```
Attempting PDF compilation from custom branch paper-branch. Reticulating splines etc...
```

**Comment by whedon** [comment]
:point_right::page_facing_up: [Download article proof](https://raw.githubusercontent.com/openjournals/joss-papers/joss.02166/joss.02166/10.21105.joss.02166.pdf) :page_facing_up: [View article proof on GitHub](https://github.com/openjournals/joss-papers/blob/joss.02166/joss.02166/10.21105.joss.02166.pdf) :page_facing_up: :point_left:

**Comment by kyleniemeyer** [comment]
@whedon accept deposit=true from branch paper-branch

**Comment by whedon** [comment]
```
Doing it live! Attempting automated processing of paper acceptance...
```

**Comment by whedon** [comment]
🐦🐦🐦 👉 [Tweet for this paper](https://twitter.com/JOSS_TheOJ/status/1320430666115358721) 👈 🐦🐦🐦

**Comment by whedon** [comment]
🚨🚨🚨 **THIS IS NOT A DRILL, YOU HAVE JUST ACCEPTED A PAPER INTO JOSS!** 🚨🚨🚨

 Here's what you must now do:

0. Check final PDF and Crossref metadata that was deposited :point_right: https://github.com/openjournals/joss-papers/pull/1856
1. Wait a couple of minutes to verify that the paper DOI resolves [https://doi.org/10.21105/joss.02166](https://doi.org/10.21105/joss.02166)
2. If everything looks good, then close this review issue.
3. Party like you just published a paper! 🎉🌈🦄💃👻🤘

 Any issues? Notify your editorial technical team...

**Comment by kyleniemeyer** [comment]
Congrats @hvgazula on your article's publication in JOSS!

Many thanks to @gkiar and @yarikoptic for reviewing this, and @cMadan for editing.

**Comment by whedon** [comment]
:tada::tada::tada: Congratulations on your paper acceptance! :tada::tada::tada:

If you would like to include a link to your paper from your README use the following code snippets:

```
Markdown:
[![DOI](https://joss.theoj.org/papers/10.21105/joss.02166/status.svg)](https://doi.org/10.21105/joss.02166)

HTML:
<a style="border-width:0" href="https://doi.org/10.21105/joss.02166">
  <img src="https://joss.theoj.org/papers/10.21105/joss.02166/status.svg" alt="DOI badge" >
</a>

reStructuredText:
.. image:: https://joss.theoj.org/papers/10.21105/joss.02166/status.svg
   :target: https://doi.org/10.21105/joss.02166
```
This is how it will look in your documentation:

[![DOI](https://joss.theoj.org/papers/10.21105/joss.02166/status.svg)](https://doi.org/10.21105/joss.02166)

**We need your help!**

Journal of Open Source Software is a community-run journal and relies upon volunteer effort. If you'd like to support us please consider doing either one (or both) of the the following:

- Volunteering to review for us sometime in the future. You can add your name to the reviewer list here: https://joss.theoj.org/reviewer-signup.html
- Making a small donation to support our running costs here: https://numfocus.org/donate-to-joss
